src/main/java/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -189,16 +189,20 @@ public String getDatabaseDDLPath(DatabaseType db_type) {
         // The order matters!
         List<String> names = new ArrayList<>();
         if (db_type != null) {
-            names.add("ddl-" + db_type.name().toLowerCase() + ".sql");
+            if (DatabaseType.MARIADB == db_type) {
+                // common ddl for mariadb and mysql
+                names.add("ddl-mysql.sql");
```

Comment:
Instead of hardcoding the string "-mysql", you should override the `db_type`:
```
DatabaseType ddl_db_type = db_type;
if (ddl_db_db_type == DatabaseType.MARIADB) ddl_db_type = DatabaseType.MYSQL;
names.add("ddl-" + ddl_db_type.name().toLowerCase() + ".sql");
```

Reply:
_corrected_